terraform-migrate-state-tfc/step3.md

```diff
@@ -0,0 +1,63 @@
+# Migrate State to Terraform Cloud
+
+Finally, let's migrate the project to Terraform Cloud and execute the configuration remotely. The steps include:
+
+- Run `init` on your local configuration
```

Comment:
I think it's more clear if the bullets talk about what the user is accomplishing rather than the commands used; eg "Initialize your local configuration".

Reply:
```suggestion
- Initialize your local configuration
```